cli/dts/lib.deno.unstable.d.ts

```diff
@@ -206,12 +206,14 @@ declare namespace Deno {
   /** **UNSTABLE**: new API, yet to be vetted.
    *
    * Format an array of diagnostic items and return them as a single string in a
-   * user friendly format.
+   * user friendly format. If the diagnostic items is an empty array, it will
```

Comment:
i should really replace `is` with `any`, i think that would be better english?

Reply:
> If the diagnostic items are an empty array.

> If the diagnostic items is an empty array

🤔

Reply:
"If there are no diagnostics it will return an empty string."?

Reply:
sure :) https://github.com/denoland/deno/pull/10925/commits/c875b70962b5f6b65054adb1a264bac7723275cc